### DISCUSSION TITLE
onClick not working on my Next13 App

### DISCUSSION THREAD

**Discussion by Wills-dev** [vercel/next.js#55949] [Help]
### Summary

I have previous worked with next 13.4.10 app and everything worked out great but current I tried starting a new project with next 13 still, at some point I noticed the function I called on button Click isn't work, I also wasn't getting any error. I handle to just console.log('object') to be sure it wasn't from the function but still nothing came out. I have tried on different pages as well as different new app still no change. Below is an example.

### Additional information

```js
"use client";

import Image from "next/image";

export default function Home() {
  const handleClick = (e) => {
    console.log("object");
  };

return (
<button onClick={handleClick}>hello</button>
)
}
```

**Comment by icyJoseph** [comment]
Node.js version? You need to have at least Node 16.14

**Reply by Wills-dev** [reply]
I just switched to the latest node v18.18.0 and still the same outcome

**Reply by icyJoseph** [reply]
Oh no.... alright, then let me see...

You have the latest Next.js right? Did you delete .next also? just for good measure

**Reply by Wills-dev** [reply]
I haven't deleted .next
should I??

**Reply by icyJoseph** [reply]
I mean, it should not matter, but sometimes things are pre-compiled there... so just to discard a possible .next issue ~ please do, it is regenerated when you run dev or build

**Reply by Wills-dev** [reply]
I have done that but still didn't work.

**Reply by icyJoseph** [reply]
Alright, do you have a link to a repository? Does it happen in other projects you may have, with matching Node.js and Next.js versions?

**Reply by Wills-dev** [reply]
So I have worked on a couple of Next.js projects in the past and I didn't have this experience. I noticed this recent issue last week and I have tried restarting the project several times still the same out come. Below is the link to the repo 
https://github.com/Wills-dev/share_prompt

Thank you

**Reply by icyJoseph** [reply]
If I go to:

- https://stackblitz.com/github/Wills-dev/share_prompt/tree/main?file=app%2Fabout%2Fpage.jsx

And then open the about tab in a new window:

- https://parixkklqgithub-ilgj--3000--d6c42aca.local-credentialless.webcontainer.io/about

Then I can see the console.log printing out

**Reply by Wills-dev** [reply]
Yes, I tried the link you shared and it's working fine there but still isn't working locally on my end.

**Reply by icyJoseph** [reply]
It has to be something with your local setup, but mmm what... Maybe JS is disabled in the browser?

Just trying to discard any gotchas

**Reply by Wills-dev** [reply]
if I access any of the previous projects and do something similar, it works fine. I only experience this on new projects

**Reply by Wills-dev** [reply]
I think the problem could be  from the latest version of next i.e v13.5.3. I installed v13.4.10 and it's working fine.  
Thank you so much @icyJoseph , you were quite helpful today and I appreciate

**Reply by icyJoseph** [reply]
Aha, really? But that also worked on the Stackblitz deployment right? What OS have you got?

**Reply by icyJoseph** [reply]
oh no, your example had 13.5.1 😮 

No worries! I'll take a look with the very latest. I didn't see they released a new one today.

**Reply by icyJoseph** [reply]
One last thing, you don't happen to have a filter on your logs right? Did it also fail on incognito mode?

Anyhow, cheers! Have a good one.

**Reply by Wills-dev** [reply]
my OS is Ventura v13.2.1

**Reply by Wills-dev** [reply]
I didn't try on incognito mode. Maybe I will try and give you a feedback
